app/DoctrineMigrations/Version20161128084725.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Added view_mode in user config.
+ */
+class Version20161128084725 extends AbstractMigration implements ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    private function getTable($tableName)
+    {
+        return $this->container->getParameter('database_table_prefix').$tableName;
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $configTable = $schema->getTable($this->getTable('config'));
+        $this->skipIf($configTable->hasColumn('view_mode'), 'It seems that you already played this migration.');
+
+        $configTable->addColumn('view_mode', 'integer');
```

Comment:
Do you plan to handle more than 2 view mode?
If so, I'm ok with an integer and `view_mode` as field name.
Otherwise, just call it `list_view`